### DISCUSSION TITLE
Can resize observer only detect width changed ?

### DISCUSSION THREAD

**Discussion by tin80122** [quasarframework/quasar#10085] [General - Components / Directives / etc] [answered]
Hi all,

https://v1.quasar.dev/vue-components/resize-observer#introduction
through this way,
I want to only detect the element's width changed.
It seems that quasar's resize observer will detect the width and height changed both.
But I don't mind the height changed.

Is it possible?

Thanks.

**Comment by justinbeaty** [comment, marked as answer]
@tin80122 It's probably easiest just to remember the old width, and return early if it hasn't changed:

```
export default {
  data () {
    return {
      oldWidth: 0,
    }
  },

  methods: {
    onResize (size) {
      if(this.oldWidth === size.width) {
        return
      }
      this.oldWidth = size.width
      
      // do something
    
    },
  }
}
```

**Reply by tin80122** [reply]
It really helpful, thanks.
